app/views/shared/_project_title.html.erb

```diff
@@ -19,6 +19,4 @@
   <%= project.title %>
 <% end %>
 
-<% if project.archived? %>
-  <span class="status-badge <%= project.status %>"><%= project.status %></span>
-<% end %>
```

Comment:
this `span` should also load if the project is not archived.  
because we don't load this span for unarchived projects, when the user archives it, there is no label to show because we haven't added that to the HTML yet.

Thats why upon page refresh, it starts to work the second time onwards.

Reply:
If this has to be there always I wonder if we can hide it with css when it's not archived so it doesn't bloat the UI?